### ISSUE TITLE
bug: Band mask generation sometimes fails

### ISSUE THREAD

**Issue by Fersoil** [xai4space/meteors#78]
### Description

In some cases, when the band name provided is a two letter sign, the creation of the band mask fails

![image](https://github.com/user-attachments/assets/45829ebb-1e2c-4a02-9e91-6d645e25c054)
![image](https://github.com/user-attachments/assets/55d25951-c893-4cbc-a189-b5c88aae5d83)

### Reproduction steps

```bash
import meteors
import torch
image_tensor = torch.rand(5,5,5)
wavelengths = [100, 200, 300, 400, 500]
hsi = meteors.HSI(image = image_tensor, wavelengths = wavelengths)

band_mask, band_names = meteors.lime.Lime.get_band_mask("S2")
```
